components/colorarea/stories/colorarea.stories.js

```diff
@@ -51,4 +51,4 @@ export const CustomSize = Template.bind({});
 CustomSize.args = {
   customWidth: "80px",
   customHeight: "80px"
-};
\ No newline at end of file
+};
```

Comment:
😬 is my editor adding lines where it shouldn't be?

Reply:
I have my editor set to add a new line at the end of files just because that's the typical sparkbox coding standard. It shouldn't have committed that though since I didn't edit these files...